units/check_blocked_tasks.go

```diff
@@ -44,11 +43,11 @@ func makeCheckBlockedTasksJob() *checkBlockedTasksJob {
 
 // NewCheckBlockedTasksJob creates a job to run repotracker against a repository.
 // The code creating this job is responsible for verifying that the project
-// should track push events
+// should track push events. We want to limit this job to once an hour for each distro.
 func NewCheckBlockedTasksJob(distroId string, taskIDs []string) amboy.Job {
 	job := makeCheckBlockedTasksJob()
 	job.TaskIDs = taskIDs
-	job.SetID(fmt.Sprintf("%s:%s:%s", checkBlockedTasks, distroId, bson.NewObjectId().Hex()))
+	job.SetID(fmt.Sprintf("%s:%s:%s", checkBlockedTasks, distroId, time.Now().Round(time.Hour).String()))
```

Comment:
nit: I suspect `String()` is redundant, since `fmt.Sprintf` will call that implicitly.